### PR TITLE
RELENG-586: Add mozillaonline builds & signing

### PR DESCRIPTION
**Pull request by bhearsum**
This adds a few new builds (and signing tasks, where it makes sense) for:
- PRs, to make sure the builds continue to work
- Betas
- Releases

This can't be landed until https://github.com/mozilla-releng/scriptworker-scripts/pull/367 has been deployed to support the new certificates. I'm mainly opening this now to make sure the PR build succeeds.

### CODE REVIEW

File: taskcluster/fenix_taskgraph/transforms/signing.py [hunk 1]
```diff
@@ -58,6 +58,8 @@ def set_signing_type(config, tasks):
                 signing_type = "fennec-production-signing"
             elif task["attributes"]["build-type"] in ("nightly", "android-test-nightly"):
                 signing_type = "production-signing"
+            elif task["attributes"]["build-type"] in ("beta-mozillaonline", "release-mozillaonline"):
```
Comment (escapewindow) [inline taskcluster/fenix_taskgraph/transforms/signing.py:61]: It looks like if you follow my advice on the other PR, we'll adjust the config or logic for `set_signing_format`.

**Review by escapewindow — COMMENTED** [review]
I haven't done a deep dive here, but it looks sane. Does taskgraph-gen give you new builds and signing tasks? https://docs.mozilla-releng.net/en/latest/procedures/mobile_automation_setup.html?highlight=mobile#how-to-set-up-taskgraph-for-mobile

File: taskcluster/fenix_taskgraph/transforms/signing.py (hunk 1 above)
Comment (escapewindow) [reply taskcluster/fenix_taskgraph/transforms/signing.py:61]: (If this is difficult, we can certainly use a new cert type :)

**Comment by bhearsum** [comment]
> Does taskgraph-gen give you new builds and signing tasks? 

Yeah, I can send you diffs if you want. Both the task list, and the payloads look reasonable to me (tough to be certain without running the tasks, but there's nothing obviously wrong in them).

**Comment by escapewindow** [comment]
> > Does taskgraph-gen give you new builds and signing tasks?
> 
> Yeah, I can send you diffs if you want. Both the task list, and the payloads look reasonable to me (tough to be certain without running the tasks, but there's nothing obviously wrong in them).

I can take a look if you want, but we can probably proceed once we get the two PRs updated to use the `production-signing` cert type, or add new patches to deal with the new cert type's scopes.

File: taskcluster/fenix_taskgraph/transforms/signing.py (hunk 1 above)
Comment (bhearsum) [reply taskcluster/fenix_taskgraph/transforms/signing.py]: Looks like it was easy to do `by-build-type` in the kind.

**Comment by bhearsum** [comment]
@escapewindow I think this is ready for another look. Here's diffs on nightly & beta target tasks:
- https://gist.github.com/bhearsum/994040e739345144bcdfa2778d43a253
- https://gist.github.com/bhearsum/554313462f2653401c562c8c8a22fa84

**Review by escapewindow — APPROVED** [review]
Looks good, thank you!

**Review by st3fan — APPROVED** [review]
I scanned through this PR and it looks good to me as someone who doesn't know TC super well. Big 👍🏻 for making this happen!

**Pull request merged**